main.go

```diff
@@ -237,6 +229,9 @@ func (s *Session) login(ctx context.Context) error {
 				if location == "https://photos.google.com/" {
 					return nil
 				}
+				if *headlessFlag {
+					return errors.New("You cannot authenticate in -headless mode")
```

Comment:
let's make it impersonal please (no "you"). something like:
"authentication not possible in -headless mode"
Also, error messages should not begin with an uppercase. So that you can stack them, wrap around them, etc.

Reply:
Fixed